xbmc/cores/VideoPlayer/DVDSubtitles/SubtitlesStyle.h

```diff
@@ -103,14 +113,17 @@ struct subtitleOpts
 
 struct renderOpts
 {
+  // If true, it forces reconfiguration of the renderer
+  bool reconfigure = false;
```

Comment:
I don't really like this, `reconfigure` is an action you do on a certain item not really some option you pass to define how to render elements. I don't have any better suggestion atm but I guess you understand my concern here and maybe can think of a possible alternative?
Maybe convey some member relater to position so that later on the render you can decide if you should reconfigure or not based on state change

Reply:
my failure forgot to double-check, i see now libass source:
https://github.com/libass/libass/blob/1140b6b885c89d37eef13dc1f31f144e9a76a4d7/libass/ass_render_api.c#L131-L137
there is no problem if we set it at each frame, the method is noop if value is not changed
then i can remove this